src/KafkaFlow.Serializer.ProtobufNet/KafkaFlow.Serializer.ProtobufNet.csproj

```diff
@@ -13,5 +13,6 @@
 
     <ItemGroup>
         <ProjectReference Include="..\KafkaFlow.Abstractions\KafkaFlow.Abstractions.csproj"/>
+        <ProjectReference Include="..\KafkaFlow\KafkaFlow.csproj" />
```

Comment:
I think this dependency is no more needed

Reply:
Removed